src/player.cpp

```diff
@@ -10239,11 +10239,16 @@ bool player::wield(item* it, bool autodrop)
         return false;
     }
 
-    if( it->is_two_handed(*this) && !has_two_arms() ) {
-        add_msg(m_info, _("You cannot wield a %s with only one arm."),
+    if ( it->is_two_handed(*this) && !has_two_arms() ) {
+        if (it->has_flag("ALWAYS_TWOHAND")) {
+            add_msg(m_info, _("The %s can't be wielded with only one arm."),
                 it->tname().c_str());
-        return false;
```

Comment:
Why is this `return` statement gone? I think it was important. It looks as if the function now allows you to wield the item anyway, in contrast to the messages.